python/tutorials/09-persistent-fp8-matmul.py

```diff
@@ -388,6 +392,10 @@ def validate(M, N, K):
     )
 
 
+if not hasattr(torch, "float8_e4m3fn") or not is_cuda():
```

Comment:
the is_cuda check is redundant with line13?

Reply:
Sorry, I have totally forgot I have already addressed this problem! Thanks for catching that!